include/RED4ext/GameEngine.hpp

```diff
@@ -19,6 +19,14 @@ namespace Memory
 struct IAllocator;
 }
 
+enum class EEngineState : int32_t
```

Comment:
Are these the same as https://github.com/WopsS/RED4ext.SDK/blob/master/include/RED4ext/GameStates.hpp#L11-L17 or is this really different?

Reply:
Different.